src/runtime_src/tools/xclbin/CMakeLists.txt

```diff
@@ -74,7 +74,8 @@ file(GLOB XCLBINUTIL_FILES
   "FDT*.cxx"
   "FormattedOutput.cxx"
   "ParameterSectionData.cxx"
-  "Section*.cxx"
+  "Section.cxx"     # Note: Due to linking dependency issue, this entry needs to be before the other sections
+  "Section*.cxx"  
```

Comment:
Argh. so Section.cxx is looking for symbols in Section*.cxx?  This is a static linking issue, right?

Reply:
Yes it is :(.  
Lizhi encountered this issue earlier this week and this is only a temporary fix.  
I have created a task (e.g., XRT-438) to address this issue correctly post 2020.1.

Reply:
done